### ISSUE TITLE
my-badges does not find commits in the repositories of the Organizations I am a member of

### ISSUE THREAD

**Issue by andrewjswan** [my-badges/my-badges#42]
**my-badges** does not find commits in the repositories of the Organizations I am a member of.

**Comment by andrewjswan** [comment]
Yes, I saw it, but there are no repositories in Organizations that have my commits for example:
https://github.com/MPExtended/MPExtended/commits/Release-0.7/
https://github.com/MediaPortal/TitanExtended/commits/master/
https://github.com/MediaPortal/MediaPortal-1/commits/master/

**Comment by antonmedv** [comment]
Well. This is what `GET /users/{username}/repos` gives for as. 

I guess a flag for additional repos can be added as a feature?

**Comment by andrewjswan** [comment]
> Well. This is what `GET /users/{username}/repos` gives for as.
Strange, but not only my repositories are on the list:

Loading commits for `yoavain`/mediaportal-fanart-handler
Loading commits for `yoavain`/mediaportal-latest-media-handler

> I guess a flag for additional repos can be added as a feature?

If it is not possible to indicate in the selection all the repositories, including the repositories of the Organizations to which the user is a member, then any working solution would be great.
